packages/amplify-cli/src/commands/init.ts

```diff
@@ -10,6 +10,7 @@ import { initProviders } from '../init-steps/s2-initProviders';
 import { scaffoldProjectHeadless } from '../init-steps/s8-scaffoldHeadless';
 import { onHeadlessSuccess, onSuccess } from '../init-steps/s9-onSuccess';
 import { checkForNestedProject } from './helpers/projectUtils';
+import { migrationWarning } from '../init-steps/preInitMigrationWarning';
```

Comment:
It's not migration related warning. Perhaps `gen2Recommendation`?

Reply:
Good idea. Done [49a9f01](https://github.com/aws-amplify/amplify-cli/pull/13849/commits/49a9f01f2e70fd3c9b41b336b643e38f2e47d55f)